content/concepts/transports/webrtc.md

```diff
@@ -1,9 +1,150 @@
 ---
 title: "WebRTC"
-description: "WebRTC is a new specification that uses QUIC to offer an alternative to WebSocket. Conceptually, it can be considered WebSocket over QUIC.Learn about WebTransport and how it is used in libp2p."
+description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other applications."
```

Comment:
```suggestion
description: "WebRTC is a protocol for real-time communication and is used to establish connections between browsers and other nodes."
```

Would be more intuitive for me. In my mental model applications are running on top of WebRTC.